cmake/3rdParty/FindWwise.cmake

```diff
@@ -9,13 +9,33 @@
 # Wwise Install Path
 set(LY_WWISE_INSTALL_PATH "" CACHE PATH "Path to Wwise installation.")
 
-# Check for a known file in the SDK path to verify the path
+# Wwise Version
+set(WWISE_VERSION)
+set(MINIMUM_WWISE_VERSION "2021.1.1.0")
```

Comment:
Our build nodes meet should meet min version, but best to check this out in your AR branch build.

Reply:
This Gem isn't built on AR due to the Wwise SDK dependency.